src/containers/reservations/Reservations.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { useEffect } from 'react';
+import MapPage from '../../components/mapPageComponents/mapPage/index';
+import BlockTabs from '../../components/blockTabs/index';
+
+import { Helmet } from 'react-helmet';
+import { Typography } from 'antd';
+import useWindowDimensions, {
+  WindowTypes,
+} from '../../components/window-dimensions';
+import MobileMapPage from '../../components/mapPageComponents/mobileMapPage';
+import { MapProps, mapStateToProps } from '../../store';
+import { asyncRequestIsComplete } from '../../utils/asyncRequest';
+import { connect, useDispatch } from 'react-redux';
+import { getMapGeoData } from '../../components/mapPageComponents/ducks/thunks';
+import Accordion from './Accordion';
+
+const { Paragraph } = Typography;
+
+const Reservations: React.FC<MapProps> = ({ neighborhoods, blocks }) => {
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(getMapGeoData());
+  }, [dispatch]);
+
+  const { windowType } = useWindowDimensions();
+
+  switch (windowType) {
```

Comment:
How much does this file differ from the other map view?

Also naming convention for containers is `src/containers/[name]/index.tsx` which allows for import X from 'containers/[name]` without specifying it further

Reply:
This file and the other map view (the landing page) are pretty similar, but I don't know what the idiomatic way would be to abstract their common functionalities - a new component? A function? Furthermore, how much precisely should the abstraction include - the Helmet, the Mobile/Desktop switch/case, the Props interface and mapStateToProps for the respective containers?